### PR TITLE
Fix build docs for `--ci` saying it included `--use-inline-cache --save-inline-cache`

### PR DESCRIPTION
**Pull request by dchw**

### CODE REVIEW

**Review by alexcb — APPROVED** [review]

File: docs/earthly-command/earthly-command.md [hunk 1]
```diff
@@ -251,14 +251,10 @@ Also available as an env var setting: `EARTHLY_CI=true`
 In *target mode*, this option is an alias for
 
 ```
---use-inline-cache --save-inline-cache --no-output --strict
+--no-output --strict
 ```
 
-In *artifact* and *image modes* , this option is an alias for
```
Comment (alexcb) [inline docs/earthly-command/earthly-command.md:257]: nit: does `--strict` not apply?

Comment (dchw) [reply docs/earthly-command/earthly-command.md:257]: I have no idea. The old docs did not mention it, so I just rolled with it

Comment (alexcb) [reply docs/earthly-command/earthly-command.md:257]: strict applies in either case: https://github.com/earthly/earthly/blob/main/cmd/earthly/build_cmd.go#L58

**Review by alexcb — CHANGES_REQUESTED** [review]
looks good once `--strict` gets updated for both modes.

**Comment by vladaionescu** [comment]
This got forgotten - might be good to finish and merge

**Comment by alexcb** [comment]
@dchw ping

**Review by alexcb — APPROVED** [review]

**Pull request merged**